### PR TITLE
💅 Style code blocks

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Add following style to code blocks that are not mermaid:
- Scrollable if beyond maximum length or width
- Header bar
- Copy code functionality
- Syntax highlighting

Test plan: Expect to see code blocks in docs as per screenshot.

Screenshot 2023-01-11 at 3 54 49 PM

### CODE REVIEW

File: content/guides/farmlands-pos-integration.md [hunk 1]
```diff
@@ -85,257 +85,246 @@ sequenceDiagram
...
+      -H "Authorization: $jwt" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "code": "123456789",
+        "scannedBy": "merchant",
+        "merchantConfigId": "5ee168e8597be5002af7b454"
+      }'
+    ```
+
+    ```bash [Example API Response Payload]
```
Comment (tom-blake) [inline content/guides/farmlands-pos-integration.md:101]: Suggest we rename these captions:
+ Example API Request -> Request
+ Example API Response -> Response

File: components/content/ProseCode.vue [hunk 2]
```diff
@@ -49,20 +103,29 @@ onMounted(() => {
...
+  min-height: 1rem;
+}
+
+pre {
+  @apply
+    mb-0
+  ;
+}
+
+.v-enter-active {
```
Comment (tom-blake) [inline components/content/ProseCode.vue:118]: Let's inline these transition styles

File: content/guides/farmlands-pos-integration.md (hunk 1 above)
Comment (MeganSteenkamp) [reply content/guides/farmlands-pos-integration.md]: @tom-blake done 👍

File: components/content/ProseCode.vue (hunk 2 above)
Comment (MeganSteenkamp) [reply components/content/ProseCode.vue]: @tom-blake done 👍

**Review by tom-blake — APPROVED** [review]

**Pull request merged**